content/management/configuration-api/v3.4/index.mdx

```diff
@@ -4027,6 +4028,114 @@ Informs that a chatting Customer's session fields were updated. The webhook will
 
 </Section>
 
+## Configuration
+
+<Section>
+
+<Text>
+
+### `group_created`
+
+Informs that a group has been created on the license.
+
+#### Specifics
+
+|                        |                                                                                                                                                                                       |
+| ---------------------- | -------------------------------------------------------------------------------- |
+| **Action**             | `create_group`                                                                   |
+| **Push equivalent in** | [`Agent Chat API`](/messaging/agent-chat-api/v3.4/rtm-reference/#group_created)  |
+
+</Text>
+
+<Code>
+
+<CodeResponse title={'Sample webhook'}>
+
+</CodeResponse>
+
+```json
```

Comment:
Payloads should be put within the CodeResponse component, like so:

```
<CodeResponse title={'Sample webhook'}>

payload

</CodeResponse>
```

(with the proper indentation).

Currently, they're rendered as:

![image](https://user-images.githubusercontent.com/42515529/119127826-14354300-ba35-11eb-8a48-66fb30481302.png)